pkg/cli/cvd.go

```diff
@@ -106,6 +108,11 @@ func (c *cvdCreator) createCVDFromLocalBuild() ([]*hoapi.CVD, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Error building list of required image files: %w", err)
 	}
+	cvdHostPkgFilename := filepath.Join(vars.HostOut, CVDHostPackageName)
+	if _, err := os.Stat(cvdHostPkgFilename); errors.Is(err, os.ErrNotExist) {
+		return nil, fmt.Errorf("%q not found. Please run `m hosttar`.", CVDHostPackageName)
```

Comment:
Is there a risk that this file is out of date if the user is required to run `m hosttar` separately? This will remind them to run `m hosttar` the first time, but the user could run `m` later and not `m hosttar`.

Reply:
could `cvdr` run it itself?

Reply:
Good catch Cody, thanks.

Added an "up to date" logic comparing against `cvd-host_package` directory modification time. PTAL.

Gave it a try to run `m hosttar` programatically and realized that `m` and ` _trigger_build` are shell functions for which didn't find an easy way to propagate, https://docstore.mik.ua/orelly/unix3/upt/ch29_13.htm is the closer I found.